components/BookingForm/index.js

```diff
@@ -0,0 +1,158 @@
+
+    try {
+      const response = await fetch("/api/bookings/", {
+        method: "POST",
+        headers: {
+          "Content-Typ": "application/json",
+        },
+        body: JSON.stringify(bookingData),
+      });
+
+      if (response.ok) {
+        Router.push("/");
+      } else {
+        alert(
+          "Nope, didn't work, please check if you filled in everything completely"
+        );
+      }
+    } catch (error) {
+      alert("An error ocurred");
+    }
+  };
+
+  return (
+    <>
+      <StyledBackButton href="/" alt="home">
+        <FaArrowLeft />
+      </StyledBackButton>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <StyledTitle
+            placeholder="Give your training a title"
+            type="text"
+            id="title"
+            name="title"
+            value={bookingData.title}
+            onChange={handleChange}
+          />
+        </div>
+        <div>
+          <StyledBookedPerson
+            placeholder="Who is training?"
+            type="text"
+            id="bookedPerson"
+            name="bookedPerson"
+            value={bookingData.bookedPerson}
+            onChange={handleChange}
+          />
+        </div>
+        <div>
+          <StyledDate
+            type="date"
+            id="date"
+            name="date"
+            value={bookingData.date}
+            onChange={handleChange}
+          />
+        </div>
+        <div>
+          <StyledStartTime
+            type="time"
+            id="sTime"
+            name="sTime"
+            value={bookingData.sTime}
+            onChange={handleChange}
+          />
+        </div>
+        <div>
+          <StyledEndTime
+            type="time"
+            id="eTime"
+            name="eTime"
+            value={bookingData.eTime}
+            onChange={handleChange}
+          />
+        </div>
+        <div>
+          <StyledChooseArena
+            placeholder="Choose Arena"
+            id="bookedArena"
+            name="bookedArena"
+            value={bookingData.bookedArena}
+            onChange={handleChange}
+          >
+            <option value="">Choose Arena</option>
+            <option value="Dressage Arena">Dressage Arena</option>
+            <option value="Dressage Arena outside">
+              Dressage Arena outside
+            </option>
+            <option value="Jumping Arena">Jumping Arena</option>
+            <option value="Military Arena">Military Arena</option>
+            <option value="Round Pen">Round Pen</option>
+          </StyledChooseArena>
+        </div>
+        <div>
+          <StyledLabel htmlFor="singleRiderCheck">Single Rider</StyledLabel>
+          <StyledCheckbox
+            type="checkbox"
+            id="singleRider"
+            name="singleRider"
+            checked={bookingData.singleRider}
+            onChange={handleChange}
+          />
+        </div>
+        <div>
+          <BookArenaButton type="submit" onSubmit={handleSubmit}>
```

Comment:
If you have the `onSubmit` event listener on the form then we don't need to also add it to the button. 

```suggestion
          <BookArenaButton type="submit">
```